src/definition/bazel_goto_definition_provider.ts

```diff
@@ -41,33 +74,15 @@ export class BazelGotoDefinitionProvider implements DefinitionProvider {
 
     const range = document.getWordRangeAtPosition(position, LABEL_REGEX);
     const targetText = document.getText(range);
-    const match = LABEL_REGEX.exec(targetText);
-
-    const targetName = match[1];
-    // don't try to process visibility targets.
-    if (targetName.startsWith("//visibility")) {
-      return null;
-    }
 
-    const queryResult = await new BazelQuery(
-      getDefaultBazelExecutablePath(),
-      Utils.dirname(document.uri).fsPath,
-    ).queryTargets(`kind(rule, "${targetName}") + kind(file, "${targetName}")`);
+    const location = await targetToUri(targetText, Utils.dirname(document.uri));
```

Comment:
I'm not sure if this case can ever realistically get hit, but to make this refactor preserve behaviour you should return null here if location is null.

Reply:
Fixed with ternary. Let me know if you'd rather I used some other format.